x-pack/filebeat/module/sophosxg/firewall/test/sandbox.log-expected.json

```diff
@@ -0,0 +1,313 @@
+[
+    {
+        "@timestamp": "2017-01-31T14:52:11.000-02:00",
+        "event.action": "allowed",
+        "event.category": [
+            "network"
+        ],
+        "event.code": "138301618041",
+        "event.dataset": "sophosxg.firewall",
+        "event.kind": "event",
+        "event.module": "sophosxg",
+        "event.original": "device=\"SFW\" date=2017-01-31 time=14:52:11 timezone=\"IST\" device_name=\"CR750iNG-XP\" device_id=C44310050024-P29PUA log_id=138301618041 log_type=\"Sandbox\" log_component=\"Mail\" log_subtype=\"Allowed\" priority=Information user_name=\"\" src_ip= filename=\"\" filetype=\"\" filesize=0 sha1sum=\"\" source=\"\" reason=\"eligible\" destination=\"\" subject=\"\"",
+        "event.outcome": "success",
+        "event.severity": "6",
+        "event.timezone": "-02:00",
+        "event.type": [
+            "allowed",
+            "end",
+            "connection"
+        ],
+        "file.size": "0",
+        "fileset.name": "firewall",
+        "input.type": "log",
+        "log.level": "informational",
+        "log.offset": 0,
+        "observer.name": "firewall.localgroup.local",
+        "observer.product": "XG",
+        "observer.serial_number": "C44310050024-P29PUA",
+        "observer.type": "firewall",
+        "observer.vendor": "Sophos",
+        "service.type": "sophosxg",
+        "sophosxg.firewall.device": "SFW",
+        "sophosxg.firewall.device_name": "CR750iNG-XP",
+        "sophosxg.firewall.log_component": "Mail",
+        "sophosxg.firewall.log_subtype": "Allowed",
+        "sophosxg.firewall.log_type": "Sandbox",
+        "sophosxg.firewall.message_id": "18041",
+        "sophosxg.firewall.priority": "Information",
+        "sophosxg.firewall.reason": "eligible",
+        "tags": [
+            "sophosxg-firewall forwarded"
+        ]
+    },
+    {
+        "@timestamp": "2017-01-31T14:52:11.000-02:00",
+        "client.ip": "10.198.47.112",
+        "event.action": "denied",
+        "event.category": [
+            "malware",
+            "network"
+        ],
+        "event.code": "138302218042",
+        "event.dataset": "sophosxg.firewall",
+        "event.kind": "alert",
+        "event.module": "sophosxg",
+        "event.original": "device=\"SFW\" date=2017-01-31 time=14:52:11 timezone=\"IST\" device_name=\"CR750iNG-XP\" device_id=C44310050024-P29PUA log_id=138302218042 log_type=\"Sandbox\" log_component=\"Mail\" log_subtype=\"Denied\" priority=Critical user_name=\"jsmith@iview.com\" src_ip=10.198.47.112 filename=\"1.exe\" filetype=\"application/octet-stream\" filesize=153006 sha1sum=\"83cd339302bf5e8ed5240ca6383418089c337a81\" source=\"jsmith@iview.com\" reason=\"cached malicious\" destination=\"\" subject=\"\"",
+        "event.outcome": "success",
+        "event.severity": "2",
+        "event.timezone": "-02:00",
+        "event.type": [
+            "diened",
+            "connection"
+        ],
+        "file.extension": "application/octet-stream",
```

Comment:
this should be mappend to "file.mime_type" not "file.extension".  An extension would be "exe" or "png"